UnitTests/DICOMcloud.Dicom.UnitTest/DicomDatasetConvertersTests.cs

```diff
@@ -88,8 +88,11 @@ public void ConvertToXml ( )
         }
 
         [TestMethod]
-        public void ConvertToJson()
+        public void ConvertToJson(Boolean SkipLargeTags = false)
```

Comment:
Hi @nutzlastfan, I have a few questions about this:
How do you pass this with "true"? 
when I try to run this test in Visual Studio 2019 it doesn't let me run it anymore if you are passing a parameter. 

If there is a special test case or the new DICOM file require this, then probably create a separate unit test clarifying what you are trying to do

Reply:
I changed it back, i kept      DicomDataset sourceDS = DicomFile.Open ( file, FileReadOption.Default).Dataset ; in so you can manualy change the option.
I tried some other files in the test, also some dicom video files so skiplargetags
comes in handy if you only want to compare the metadata.
Next time I will make an extra test.